aries-site/src/examples/templates/dashboards/content/UpdatesAvaliable.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { NotificationItem, UpdatesFeed } from '../components';
+
+const MOCK_DATA = require('../../../../data/mockData/firmware-bundles.json');
+
+export const UpdatesAvaliable = () => {
+  const data = MOCK_DATA['firmware-bundles'].items;
+  const [update, setUpdate] = useState([]);
```

Comment:
this represents multiple things and should be plural. As is now, I'd expect this to represent a single update, as opposed to multiple.

```suggestion
  const [updates, setUpdates] = useState([]);
```